src/docs/release/breaking-changes/parent-data-widget-generic-type.md

```diff
@@ -0,0 +1,119 @@
+---
+title: The generic type of ParentDataWidget changed to ParentData
+description: The ParentDataWidget is now bound to the ParentData type.
+---
+
+# Generic type of `ParentDataWidget` changed to `ParentData`
+
+
+## Context
+
+Prior to this change a `ParentDataWidget` was bound to a specific `RenderObjectWidget` type as
+ancestor. For example, a `Positioned` widget could only be used within a `Stack` widget. With this
+change, a `ParentDataWidget` can be used with any `RenderObjectWidget` type as ancestor as long as
+the `RenderObject` of said `RenderObjectWidget` sets up the correct `ParentData` type. In this new
+world, the `Positioned` widget can be reused with a hypothetical new `SuperStack` widget.
+
+
+## Description of change
+
+The generic type argument of `ParentDataWidget` has been changed from `RenderObjectWidget` to
+`ParentData`, and a new debug property `debugTypicalAncestorWidgetClass` has been added to
```

Comment:
Add commas before and after `debugTypicalAncestorWidgetClass`. (And thanks for code fonting!)